### PR TITLE
WIP: Restyle core/autoload/log.gd

### PR DESCRIPTION
**Pull request by TheDuriel**
Todo: Some vars need better names and types. Assess dead? code.

### CODE REVIEW

File: core/autoload/log.gd [hunk 1]
```diff
@@ -129,76 +128,90 @@ var dd_filter = {
...
 					else:
 						found = -1
+					
 					break
+	
 	return found
 
+
+	# TODO: Type and rename these vars
 func print_fd(id, s):
```
Comment (MolbOrg) [inline core/autoload/log.gd]: suggested name print_fd - print_filter_message

File: core/autoload/log.gd [hunk 2]
```diff
@@ -1,10 +1,12 @@
 extends Node
+"""
+	Log
+"""
 
 #verbose levels - -3:-1 errors, 0 - info, 1-3 - verbose info, 4-6 - debug
-var dd_verbosity = 4
-# var dd_verbosity = 9
+export(int, -1, 6) var _dd_verbosity: int = 4
```
Comment (MolbOrg) [inline core/autoload/log.gd]: not sure the reason for export here, it not a node which is shown in tree or editor

File: core/autoload/log.gd [hunk 3]
```diff
@@ -129,76 +128,90 @@ var dd_filter = {
...
+	}
+
 
-func test_fd(id, s):
-	var found = 1
-	if dd_filter.has(id):
-		var filter = dd_filter[id]
-		if filter.size() > 0:
+	# TODO: Type and rename these vars
+func test_fd(id, s) -> int:
```
Comment (MolbOrg) [inline core/autoload/log.gd]: test_fd - tests if a message should be printed at the current verbosity level, checks, if the message and debug_id are present in the list to be notified later, if debug message is fully registered. To take decision if should it stay and which debug level is more appropriate

File: core/autoload/log.gd [hunk 4]
```diff
@@ -129,76 +128,90 @@ var dd_filter = {
...
 			print("%-12s:: " % id, s)
-# 		-1:
+		
+#		-1:
 # 			pass #found but no need to print for enabled/verbosity level reasons
-		2:
-			print(id, "***", s)
 
 
+	# TODO: Is this code dead? If so, remove it.
```
Comment (MolbOrg) [inline core/autoload/log.gd]: yes, this can be fully removed, testing changing leftovers

File: core/autoload/log.gd (hunk 2 above)
Comment (TheDuriel) [reply core/autoload/log.gd]: It's an easy way to indicate that this value can be edited. The fact that the class is a Node at all, is a separate problem. If you can confirm that its not used as a node, we can change that.

File: core/autoload/log.gd (hunk 1 above)
Comment (TheDuriel) [reply core/autoload/log.gd]: Refactored. What are the arguments?

File: core/autoload/log.gd (hunk 3 above)
Comment (TheDuriel) [reply core/autoload/log.gd]: Renamed. Variables need new names and typing, see print_fd.

File: assets/themes/MainMenu.tres [hunk 5]
```diff
@@ -28,28 +28,28 @@ bg_color = Color( 0.105882, 0.105882, 0.105882, 1 )
 [sub_resource type="StyleBoxFlat" id=6]
```
Comment (MolbOrg) [inline assets/themes/MainMenu.tres]: godot noise, no need to commit

File: core/autoload/log.gd (hunk 2 above)
Comment (MolbOrg) [reply core/autoload/log.gd]: it isn't used as a node, it less relevant/important, just was pointing that

File: assets/themes/MainMenu.tres (hunk 5 above)
Comment (TheDuriel) [reply assets/themes/MainMenu.tres]: It'll keep making those changes until they're merged. The ID's are out of alignment on master. I'll make a separate PR with resource ID cleanup later.

File: core/autoload/log.gd (hunk 1 above)
Comment (MolbOrg) [reply core/autoload/log.gd]: id : String, s : String

**Comment by TheDuriel** [comment]
We can leave things as is for now. We're redoing the logger anyways.

Merge in master and resolve the conflicts, then we can merge.

**Pull request merged**